stories/robots/SupportedRobotScene.tsx

```diff
@@ -0,0 +1,50 @@
+import { Canvas } from "@react-three/fiber"
+import React, { useCallback } from "react"
+import type { Group } from "three"
+import { Mesh, MeshBasicMaterial, SphereGeometry } from "three"
+import { PresetEnvironment, SupportedRobot } from "../../src"
+import { rapidlyChangingMotionState } from "./motionState"
+import { OrbitControlsAround } from "./OrbitControlsAround"
+
+/**
+ * Renders a single robot model with orbit controls targeting it
+ * for storybook demonstration purposes
+ */
+export function SupportedRobotScene(
+  props: React.ComponentProps<typeof SupportedRobot>,
+) {
+  const flangeRef: React.RefCallback<Group> = useCallback((node) => {
```

Comment:
This is also a nice example on how you would add a visual tool to the robot. We could add this to the general robot docs

Reply:
yuh! though this particular approach isn't very r3f friendly, it'd probably be better to find a way that fits into the component tree